target/linux/ath79/image/generic.mk

```diff
@@ -314,6 +331,27 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/belkin_f9j1108-v2
+  SOC := qca9558
+  DEVICE_VENDOR := Belkin
+  DEVICE_MODEL := F9J1108v2 (AC1750 DB Wi-Fi)
+  DEVICE_PACKAGES += kmod-kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport
```

Comment:
`kmod-kmod-ath10k-ct` is not a valid package name, also it is not necessary to list the kmod at all, listing the package `ath10k-firmware-qca988x-ct` alone will select the kmod because it is a dependent package

so do

`  DEVICE_PACKAGES += ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport`

Reply:
OK, thanks